ui/userguide/App.jsx

```diff
@@ -10,8 +12,11 @@ const App = () => (
     <div className="card">
       <UserGuideHeader />
       <UserGuideBody />
-      <UserGuideFooter />
     </div>
+    <div className="card my-3">
+      <PerfherderUserGuide />
+    </div>
+    <UserGuideFooter />
```

Comment:
This element has the `card-footer` class, meaning that it should have been placed in a node tree that is `card`-based.
Hopefully, removing that class should suffice.

Reply:
![image](https://user-images.githubusercontent.com/63001299/94445564-41846680-01b0-11eb-9f5d-06361c8d9572.png)
It will look like this. What do you think?